### PR TITLE
sale_quotation_number: Migration to v10.0

### PR DESCRIPTION
**Pull request by DarkoNikolovski**

### CODE REVIEW

File: sale_quotation_number/__manifest__.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 {
     'name': 'Sale Quotation Numeration',
-    'version': '8.0.1.1.0',
+    'version': '10.0.1.1.0',
     'category': 'Sales Management',
     'summary': "Different sequence for sale quotations",
     'description': """
```
Comment (eLBati) [inline sale_quotation_number/__manifest__.py:30]: Please move module description to README file, following the template
https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst
and replacing 'openerp' with 'odoo'

**Review by eLBati — CHANGES_REQUESTED** [review]
Thanks!
Just a remark

**Review by antoniocanovas — CHANGES_REQUESTED** [review]
Tested in #ocacodesprint 
It would be recommended:
- Include the README
- Include a field in sale order to keep the original quotation number.
- Allow searching by default for both codes in "quotations view" once it's confirmed.

**Comment by rousseldenis** [comment]
Supersedes by https://github.com/OCA/sale-workflow/pull/591

**Pull request closed without merging**